### PR TITLE
Update NodeExpandSecretRef comment for beta 

### PR DESCRIPTION
**Pull request by humblec**
#### What type of PR is this?

/kind cleanup
/kind feature

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:

This feature has been promoted to beta by #115621 
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by humblec** [comment]
/sig storage
cc @msau42

File: staging/src/k8s.io/api/core/v1/types.go [hunk 1]
```diff
@@ -1834,9 +1834,10 @@ type CSIPersistentVolumeSource struct {
 	// nodeExpandSecretRef is a reference to the secret object containing
 	// sensitive information to pass to the CSI driver to complete the CSI
 	// NodeExpandVolume call.
-	// This is an alpha field and requires enabling CSINodeExpandSecret feature gate.
+	// This is a beta field which is enabled default by CSINodeExpandSecret feature gate.
```
Comment (msau42) [inline staging/src/k8s.io/api/core/v1/types.go:1837]: Also update the internal types.go: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L1752

Comment (humblec) [reply staging/src/k8s.io/api/core/v1/types.go:1837]: done.. thanks!

File: pkg/apis/core/types.go [hunk 2]
```diff
@@ -1749,9 +1749,10 @@ type CSIPersistentVolumeSource struct {
 	// NodeExpandSecretRef is a reference to the secret object containing
 	// sensitive information to pass to the CSI driver to complete the CSI
 	// NodeExpandVolume call.
-	// This is an alpha field and requires enabling CSINodeExpandSecret feature gate.
+	// This is a beta field which is enabled default by CSINodeExpandSecret feature gate.
```
Comment (thockin) [inline pkg/apis/core/types.go:1752]: You could just strike this line :)

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**